src/js/dataMapping/awardsv2/awardOverview.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * awardOverview.js
+ * Created by Jonathan Hill 11/26/19
+ **/
+
+// data for award overview recipient section
+// financial assistance awards are redactedDueToPII, agreggatedByState (Multiple Recipients),
+// agreggatedByCounty (Multiple Recipients), aggregatedByCountry (Multiple Recipients)
```

Comment:
what object is this mapping to?  Appears to be placeOfPerormance. Would be helpful to standardize our comments in these DataMapping sections to specify where the keys exist.